Tools/wasm/README.md

```diff
@@ -31,12 +33,18 @@ podman run --rm -ti -v $(pwd):/python-wasm/cpython:Z quay.io/tiran/cpythonbuild:
 docker run --rm -ti -v $(pwd):/python-wasm/cpython quay.io/tiran/cpythonbuild:emsdk3
 ```
 
+Note that when you exit the container, all work will be lost.
+See the `podman` or `docker` docs for how to preserve the container environment
+across runs.
+
 ### Compile a build Python interpreter
 
+From within the container, run the following commands:
+
 ```shell
 mkdir -p builddir/build
 pushd builddir/build
-../../configure -C
+../../cpython/configure
```

Comment:
The old path was correct. The new path won't work.

Reply:
No, I don't think it is so; the old path is wrong.

But let's be more explicit and use the absolute path from within the container.

Reply:
```console
$ docker run --rm -ti -v $(pwd):/python-wasm/cpython quay.io/tiran/cpythonbuild:emsdk3
Setting environment variables:
PATH = /emsdk:/emsdk/upstream/emscripten:/emsdk/node/14.18.2_64bit/bin:/usr/lib/ccache:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
root@27c144c9747d:/python-wasm# pwd
/python-wasm
root@27c144c9747d:/python-wasm# ls
README.md   build-python-build.sh               build-python-emscripten-node.sh  clean-host.sh  fetch-python.sh        run-python-node.sh  test-emscripten-node.sh
browser-ui  build-python-emscripten-browser.sh  build-python-wasi.sh             cpython        run-python-browser.sh  run-python-wasi.sh  test-wasi.sh
root@27c144c9747d:/python-wasm# mkdir -p builddir/build
root@27c144c9747d:/python-wasm# pushd builddir/build
/python-wasm/builddir/build /python-wasm
root@27c144c9747d:/python-wasm/builddir/build# ls ../../configure
ls: cannot access '../../configure': No such file or directory
root@27c144c9747d:/python-wasm/builddir/build# ls ../../cpython/configure
../../cpython/configure
```

Reply:
I'm marking this as resolved, since the new path is correct ;)